### DISCUSSION TITLE
Bold and/or Colored usernames in chat option

### DISCUSSION THREAD

**Discussion by nypocalypse** [night/betterttv#6051] [Ideas]
Is there a chance we could get a toggle for bold and/or colored @mentions/usernames mentioned like in the photo? Ignore the "mentions you" thing since that's not what I'm here for. 

![bttvsuggest](https://user-images.githubusercontent.com/126705645/228279254-e540f9b7-f0f8-4c86-929f-122ecb3de106.png)

**Comment by night** [comment]
The chat you are seeing is not rendered by BetterTTV, but rather by a different extension. You may need to disable your other extensions to revert this unwanted change.

**Reply by nypocalypse** [reply]
I'm aware the chat is not by BTTV. I was asking if we could get the option to have colored usernames such as in the photo, which I included as an example since describing it with words was not working for me.
